app/src/androidTest/java/net/skyscanner/backpack/text/BpkLinksSpanTests.kt

```diff
@@ -0,0 +1,47 @@
+package net.skyscanner.backpack.text
+
+import android.graphics.Color
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.widget.TextView
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.skyscanner.backpack.BpkSnapshotTest
+import net.skyscanner.backpack.createThemedContext
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class BpkLinksSpanTests : BpkSnapshotTest() {
+
+  private val link = "https://backpack.github.io/"
+  private val handler = { _: String ->
+  }
+
+  private val textView = TextView(testContext).apply { setBackgroundColor(Color.WHITE) }
+
+  @Before
+  fun setup() {
+    setDimensions(20, 50)
+  }
+
+  @Test
+  fun screenshotTestPrimaryColorSpan_Default() {
+    textView.text = "Test"
```

Comment:
What is this test testing? Seems to be texting just the normal text without span, if so we can remove it as the text component already tests this.

Reply:
Done